### DISCUSSION TITLE
Question about uploading files

### DISCUSSION THREAD

**Discussion by runqiL** [DeNardoLab/BehaviorDEPOT#14] [Q&A] [answered]
Hi! I'm trying to use BehaviorDEPOT to analyze some behavioral data. I was wondering for running single session, what does "one paired DLC output and video file" mean? What kind of DLC output file is it asking for? Thank you!

**Comment by cjgabrie** [comment, marked as answer]
Hello! Sorry for the late reply, and thanks for trying out BehaviorDEPOT!

The 2 inputs for analyzing a session are:
1) the video file
2) the DLC/SLEAP-based analysis of the video file (saved in either H5 or CSV formats)

The output file from DLC/SLEAP is the pose estimation of the animal for your project. It should be a file containing estimates of each tracked point for every frame in the video.
